source/contributing.html.slim

```diff
@@ -8,4 +8,10 @@ section
     p
       | If you have an idea to improve RSpec or find a bug, you are encouraged
         to file a pull request or issue with the appropriate repository on Github.
+    p
+      | See&nbsp;
+      = link_to 'rspec-dev', 'https://github.com/rspec/rspec-dev'
+      | &nbsp;for how to develop on and test RSpec, as well as any&nbsp;
+      = link_to 'DEV-README.md', '(https://github.com/rspec/rspec-core/blob/master/DEV-README.md'
+      | &nbsp;that may be in that repository.
```

Comment:
Yeah, I did. I looked at repos partial for how you handled the spacing and saw the nbsp.  I haven't used slim so just did what looked like the most consistent thing.  About to push up a fix.